workers/unity/Assets/Gdk/Android/Editor/AndroidSpatialRunner.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Net.Sockets;
+using UnityEditor;
+using UnityEngine;
+using Debug = UnityEngine.Debug;
+
+namespace Improbable.Gdk.Android.Editor
+{
+    internal class AndroidSpatialRunner
+    {
+        private static readonly string ProjectDirectory = Path.Combine(Application.dataPath, "..", "..", "..");
+        private const string ProcessName = "spatial";
+        private const string ProcessParams = "local launch";
+        private const string Config = "android_launch.json";
+
+        [MenuItem("Improbable/Run Spatial for Android")]
+        public static void RunAndroidSpatial()
+        {
+            Process.Start(GetProcessStartInfo());
+        }
+
+        private static string GetLocalIpAddress()
```

Comment:
return `IPAddress` instead of `string`. This will make it simpler to use, if we need it anywhere else and for the current use case the conversion to string will happen automatically